setup.py

```diff
@@ -1,12 +1,19 @@
 from platform import python_implementation
 from setuptools import setup, Extension
 from sys import version_info
+import sys
 from wheel.bdist_wheel import bdist_wheel
 
+def is_freethread():
```

Comment:
Is there any reason why this check should not simply use `sysconfig.get_config_var("Py_GIL_DISABLED")` as suggested by [the official docs](https://docs.python.org/3/howto/free-threading-extensions.html#limited-c-api-and-stable-abi)?